tests/testthat/test-shinytest2-tm_a_mmrm.R

```diff
@@ -0,0 +1,526 @@
+  {
+    skip_if_too_deep(5)
+
+    app_driver <- app_driver_tm_a_mmrm()
+    app_driver$expect_no_shiny_error()
+    app_driver$expect_no_validation_error()
+
+    null_text <- app_driver$active_module_element_text("null_input_msg")
+
+    testthat::expect_match(null_text, "Please first specify 'Model Settings' and press 'Fit Model'")
+
+    app_driver$stop()
+  }
+)
+
+testthat::test_that(
+  paste0(
+    "e2e - tm_a_mmrm: Module initializes with specified label, aval_var, paramcd,",
+    "visit_var, cov_var, arm_var, buckets, combine_comp_arms, id_var,",
+    "cor_struct, weights_emmeans, conf_level, method, parallel and output_function"
+  ),
+  {
+    skip_if_too_deep(5)
+    app_driver <- app_driver_tm_a_mmrm()
+
+    testthat::expect_equal(app_driver$get_text("#teal-main_ui-root-active_tab > li.active > a"), "MMRM")
+
+    testthat::expect_equal(app_driver$get_active_module_input("aval_var-dataset_ADQS_singleextract-select"), "AVAL")
+
+    testthat::expect_equal(
+      app_driver$get_active_module_input("paramcd-dataset_ADQS_singleextract-filter1-vals"),
+      "FKSI-FWB"
+    )
+
+    testthat::expect_equal(app_driver$get_active_module_input("visit_var-dataset_ADQS_singleextract-select"), "AVISIT")
+
+    testthat::expect_null(app_driver$get_active_module_input("cov_var-dataset_ADQS_singleextract-select"))
+
+    testthat::expect_equal(app_driver$get_active_module_input("arm_var-dataset_ADSL_singleextract-select"), "ARM")
+
+    testthat::expect_equal(
+      app_driver$get_active_module_input("buckets"),
+      list(
+        Ref = list("A: Drug X"),
+        Comp = list("B: Placebo", "C: Combination")
+      )
+    )
+
+    testthat::expect_false(app_driver$get_active_module_input("combine_comp_arms"))
+
+    testthat::expect_equal(app_driver$get_active_module_input("id_var-dataset_ADQS_singleextract-select"), "USUBJID")
+
+    testthat::expect_equal(app_driver$get_active_module_input("weights_emmeans"), "proportional")
+
+    testthat::expect_equal(app_driver$get_active_module_input("cor_struct"), "unstructured")
+
+    testthat::expect_equal(app_driver$get_active_module_input("conf_level"), "0.95")
+
+    testthat::expect_equal(app_driver$get_active_module_input("method"), "Satterthwaite")
+
+    testthat::expect_true(app_driver$get_active_module_input("parallel"))
+
+    testthat::expect_equal(app_driver$get_active_module_input("output_function"), "t_mmrm_lsmeans")
+
+    app_driver$stop()
+  }
+)
+
+testthat::test_that("e2e - tm_a_mmrm: Click on fit model shows table for default selection.", {
+  skip_if_too_deep(5)
+  app_driver <- app_driver_tm_a_mmrm()
+  app_driver$click(selector = app_driver$active_module_element("button_start"))
+  app_driver$expect_no_validation_error()
+
+  table <- app_driver$get_active_module_tws_output("mmrm_table")
+  col_val <- app_driver$get_active_module_input("buckets")
+  testthat::expect_true(all(unlist(col_val, use.names = FALSE) %in% colnames(table)))
+  testthat::expect_gte(nrow(table), 25)
```

Comment:
```suggestion
  testthat::expect_equal(nrow(table), 25)
```
Why not equal assertion?

Reply:
[43cd167](https://github.com/insightsengineering/teal.modules.clinical/pull/1131/commits/43cd1677864c4707e4b563e8ef98051a3bcf1bcf)